packages/terra-brand-footer/src/terra-dev-site/doc/example/BrandFooterExampleCommon.scss

```diff
@@ -0,0 +1,30 @@
+:local {
+  .content-left-anchor {
+    background-color: transparent;
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .content-left-graphics {
+    border: 0;
+    height: 40px; // setting this explicity for IE11
+    vertical-align: middle;
+    width: 40px; // setting this explicity for IE11
+  }
+
+  .content-left-terratext {
+    letter-spacing: 1px;
+  }
+
+  .content-left-uitext {
+    font-weight: 100;
+    letter-spacing: 1px;
```

Comment:
we can also implement this using a variable like @content-letter-spacing: 1px .  and then assign the variable  to letter-spacing.

Reply:
I implemented this by separating the styles as follows:
`.content-left-text {
    letter-spacing: 1px;
}

.weighted-text {
    font-weight: 100;
}`
and then by adding both these classnames to the element